tests/setupTests.js

```diff
@@ -4,3 +4,6 @@
  *
  * Run for each test to guarantee a clean state
  */
+
+// @see https://github.com/testing-library/jest-dom
+import "@testing-library/jest-dom";
```

Comment:
This probably won't work:
- try using a jest-dom matcher in a test, for instance `toContainHTML` (https://github.com/testing-library/jest-dom#tocontainhtml) => TypeScript will complain
- check how it's setup in vulcan npm